runtime/interface.go

```diff
@@ -41,93 +43,110 @@ type Block struct {
 type ResolvedLocation = sema.ResolvedLocation
 type Identifier = ast.Identifier
 type Location = common.Location
+type AddressLocation = common.AddressLocation
 
-type Interface interface {
-	// ResolveLocation resolves an import location.
-	ResolveLocation(identifiers []Identifier, location Location) ([]ResolvedLocation, error)
-	// GetCode returns the code at a given location
-	GetCode(location Location) ([]byte, error)
```

Comment:
this returns the contract code right?

Reply:
Correct. What is the replacement for this?

Reply:
ContractCode(address AddressLocation)?